department-of-veterans-affairs/va.gov-cms
# VACMS-9410: Clear event registration info when toggling states.

## Description

Relates to #9410

## QA steps
1) Go to https://pr9593-z2tm41jng73mcitwl40kpzruinndwc2h.ci.cms.va.gov/node/add/event
2) Fill out all required fields
3) Click the box for "Include registration information"
4) Select an option other than "none" for Call to action.
5) Put in a URL
6) Change the Call to Action select to "- None -"
7. +Save the node.
- [x] +Validate that no url is showing on node view. 
8. +Edit the node. and change the CTA to an option other than -none-.
- [x] 7) Confirm that the URL is now blank
8) Select an option other than "none" for Call to action.
9) Put in a URL
10) Uncheck the "Include registration information" checkbox
11) Check the "Include registration information" checkbox
- [x] 12) Confirm that the Call to Action is now set to "- None -" and the URL is empty.

13) +Edit the node.
14) +Set the call to action to "Apply".
15) +Enter a url.
16) +save the node
17) +Edit the node
18) +Uncheck the "Include registration information"
19) +save the node
- [ ]  20) +Validate that the node saves with no values for the url or its type. 

### Definition of Done


### Select Team for PR review
